docs/default.nix

```
@@ -192,27 +201,48 @@ let
                   else
                     maintainer.name;
 
-                joinItems =
-                  items:
-                  if builtins.length items <= 2 then
-                    builtins.concatStringsSep " and " items
-                  else
-                    builtins.concatStringsSep ", " (
-                      lib.dropEnd 1 items ++ [ "and ${lib.last items}" ]
-                    );
-
                 renderedMaintainers = joinItems (map renderMaintainer maintainers);
 
                 maintainersText =
                   if maintainers == [ ] then
-                    "This module has no [dedicated maintainers](../../modules.md#maintainers)."
+                    ''
+                      This module has no [dedicated maintainers](../../modules.md#maintainers).
+                    ''
+                  else
+                    ''
+                      This module is maintained by ${renderedMaintainers}.
+                    '';
```

just a small nit, but there's no reason to make these multi-line strings
```suggestion
                    "This module has no [dedicated maintainers](../../modules.md#maintainers)."
                  else
                    "This module is maintained by ${renderedMaintainers}.";
```